### DISCUSSION TITLE
Re-Connect funktioniert bei mir aktuell nicht (mehr)

### DISCUSSION THREAD

**Discussion by emufan** [SukramJ/aiohomematic#1270] [Q&A]
Vorab: Ich habe keine Logs und dieser Eintrag ist erst einmal um zu fragen, was ich wo als Log finde, was hier hilfreich wäre im Fall der Fälle.

Problem: Wenn HA weiterläuft (USV) und die CCU wegen Stromausfall wegbricht und nach Stromausfall wieder hochfährt, klappt es bei mir nie mehr, dass HA hier die Verbindung wiederherstellt und weitermacht.

Aufgetreten in den 3 der letzten 3 Fällen (haben hier gerade im Ort Stromprobleme).

Es erscheinen stattdessen immer nur diese Ping/Pong-Meldungen oder Bidcos nicht erreichbar, usw. 

Ich bin mir sicher, dass das früher ging. Evtl. seit dem Einbau der Ping/Pong-Meldung nicht mehr - keine Ahnung.

Es hilft dann immer nur HA-Re-Start oder vorhin sogar erst ein CCU-Neustart gefolgt von einem HA-Restart.

Woran könnte es liegen und/oder was kann ich nächstes Mal sichern (man ist für solch Fälle ja nicht im Dauer-Debug-Mode) und hier posten kann?

**Comment by danielperna84** [comment]
> Es erscheinen stattdessen immer nur diese Ping/Pong-Meldungen

Kannst du das etwas präzisieren? Denn wenn die CCU das Pong zum vom HA gesendeten Ping als Antwort schickt, dann ist der _Push_-Kanal in Richtung HA ja am laufen. Oder meintest, dass du zu diesem Thema Fehlermeldungen bekommst, und das Ping-Pong __nicht__ funktioniert?

**Reply by emufan** [reply]
Die CCU sendet ja. Nur meint die Integration mW, dass es nicht die CCU ist, die sie erwartet. Ich meine Sukram hat da mal was eingebaut (2027.7.1 ff) um irgendetwas abzufangen. Und das war der Zeitpunkt bei dem eine Ping/Pong-Fehlermeldung.

Nur ist es hier ja weiter/wieder die vorherige, einzige CCU und dennoch verarbeitet das HA die Updates mE nicht mehr, zumindest hatten viele Sensoren dann stundenlang keinen neuen Status und meine HA-Alarme (Kühlschrankzwischenstecker ohne update für x Minuten oder Stromverbrauch Gefirerschrank unter x W für y Minuten) finden alle an zu feuern. Jedes Mal. Und da konnte ich mich dann tatsächlich nicht um mehr kümmern als alles wieder in Gang zu bringen und die Alarme wegzubekommen.

Die Core-Integration hatte da ja auch Probleme. Aber mit dem Hilfsprogramm und automatisiertem Integrationsneustart war ja dennoch alles prima. Mit der neuen Integration war das ja viel besser und Re-Connects waren direkt da. Nur halt jetzt nicht mehr.

**Reply by SukramJ** [reply]
> CCU und dennoch verarbeitet das HA die Updates mE nicht mehr,

Da der Reconnect (bei mir) nach wie vor funktioniert, solltest Du dir mal die CCU Logs anschauen, vielleicht findet sich da ein Hinweis warum die CCU nicht sendet.

**Reply by emufan** [reply]
Warum meinst Du, dass meine CCU nicht sendet? Würden dann die Ping/Pong Mismatch Meldungen im HA kommen, bzw. woher würden die dann kommen?

**Reply by SukramJ** [reply]
Schau einfach ins Log.
Wenn Ping gesendet und kein Pong zurückkommt -> Ping/Pong Mismatch.

**Reply by emufan** [reply]
O.k. Dann gucke ich nächstes Mal dort. Verstehe es zwar weiterhin nicht, aber um zu wissen wo ich gucken muss, dafür war ja die Anfrage.

Dieses Mal bekomme ich das nicht mehr zusammen, wann welches System neu gestartet wurde und welche Meldungen kamen, weil dann welches System (gewollt, weil Neustart) weg war, usw.

Und zur Not baue ich mir die alte Systemvariablen-Methode mit Integration-Auto-Neustart wieder ein, wenn die neue Integration das nicht (mehr) stabil hinbekommt, wie es zu Beginn war. Denn wie geschrieben, hat der Reconnect in den letzten 3 von 3 Stromausfall-Fällen definitiv nicht geklappt. Das kann ja nicht jedes Mal Zufall gewesen sein. Mag ja am CCU-Hardreset wg. Stromausfall liegen (wie geschrieben HA ist wg. USV weiterhin an).

**Reply by SukramJ** [reply]
> das nicht (mehr) stabil hinbekommt, wie es zu Beginn war.

Der Mechanismus hat sich nicht geändert.

**Reply by danielperna84** [reply]
Was genau sind denn die Logs auf der Home Assistant Seite zu der Zeit wo das passiert? Da sollte doch irgendwas in _Warning_ oder _Error_ geloggt werden. Fangen wir doch erst mal damit an.

**Reply by emufan** [reply]
Wie gesagt, beim nächsten Mal gucke ich. Die letzten Male war es erst mal mal wichtiger die Alarme auszubekommen und das System wieder funktionierend.

> Der Mechanismus hat sich nicht geändert.

Mag sein, ging dennoch früher ohne Probleme und die letzten 3 von 3 Malen halt nicht. Ist so. ;-) So etwas ist ja auch nicht, was man täglich testet. Von daher kann ich nicht sagen, seit wann es (bei mir) nicht mehr geht. Dennoch habe ich hier ja seit Einbau der Ping/Pong-Meldungen schon einige Meldungen gelesen, dass irgendwas nicht mehr ging, Du Neustarts empfohlen hast, usw. Also wird sich ja schon irgendetwas geändert haben. Ob das nun mit meinem Problem zu tun hat, weiß ich natürlich 0,0.

Warten wir also den nächsten Stromausfall ab. Mag nur zum Testen ungern den CCU Stecker hart ziehen.

**Reply by SukramJ** [reply]
Um es noch mal klar zu stellen: 
Ping / Pong hat keine Auswirkung auf den ReConnect. Das ist eine unabhängige Metrik die weitere Probleme im System aufzeigen soll.
Spekulation hilft hier nicht. Das einzige was zählt sind die Logs von HA, CCU, ... .

Ein Stromausfall ist auch was anderes als eine heruntergefahrene oder neustartende CCU. Zumindest bei mir wären noch weitere Netzwerkkomponenten betroffen, die teilweise Minuten für einen Start brauchen.

> Es hilft dann immer nur HA-Re-Start oder vorhin sogar erst ein CCU-Neustart gefolgt von einem HA-Restart.

In deinem Fall würden mich vor allem die CCU Logs interessieren, da der PING scheinbar gesendet wird, aber die CCU keinen PONG senden kann, bzw. HA keinen empfängt.
